### PR TITLE
[14.0] dotfiles update needs manual intervention

### PR DESCRIPTION
**Pull request by sbidoul**
Dear maintainer,

After updating the dotfiles, `pre-commit run -a`
fails in a manner that cannot be resolved automatically.

Can you please have a look, fix and merge?

Thanks,

### CODE REVIEW

**Comment by olaf-wagner** [comment]
Hi Stephane,

the content of the Branch 14.0 has been created by Pedro afaics. We're completely new here, as is the vertical-rental repo, and have only committed some initial modules to the 12.0 branch. Migration to 13.0 and 14.0 is pending and nothing has been committed there yet from our side. So I don't really understand why the check script fails with
```
forbidden files..........................................................Failed

98 - hook id: forbidden-files
99  - exit code: 1
100
101 found forbidden files; remove them
102
103 README.md.rej
104
```
Any concreate hints what we should do to help?

File: README.md.rej [hunk 1]
```diff
@@ -0,0 +1,5 @@
+diff a/README.md b/README.md	(rejected hunks)
+@@ -1,2 +1,2 @@
+-[![Runbot Status](https://runbot.odoo-community.org/runbot/badge/flat//14.0.svg)](https://runbot.odoo-community.org/runbot/repo/github-com-oca-vertical-rental-)
```
Comment (pedrobaeza) [inline README.md.rej]: @sbidoul this is weird that the runbot id is not put her and why it fails to merge. Maybe you miss something when running the script?

Comment (sbidoul) [reply README.md.rej]: The problem comes from the copier template that vendors maintainer tools as a git submodule that needs updating whenever a repo is added. https://github.com/OCA/oca-addons-repo-template/issues/87

In the meantime, we can just fix the runbot URL manually here (vertical-rental runbot id is 298)

Comment (pedrobaeza) [reply README.md.rej]: Thanks for the clarifications, Stéphane. Let's see if my colleague knows about other strategy.

**Pull request merged**